addons/project/models/project.py

```diff
@@ -1659,6 +1697,9 @@ def write(self, vals):
             # reset kanban state when changing stage
             if 'kanban_state' not in vals:
                 vals['kanban_state'] = 'normal'
+        # portal_user_ids change: update user_ids
+        if 'portal_user_ids' in vals and 'user_ids' not in vals:
+            vals['user_ids'] = vals['portal_user_ids']
```

Comment:
@odony I'm not sure I understand your comment over here, could you elaborate ? :)

Reply:
> @odony I'm not sure I understand your comment over here, could you elaborate ? :)

I meant the obvious: that boilerplate code (`_search_portal_user_ids` + overrides of create and write for `portal_user_ids`) should not be necessary, as that's the default behavior of a related field ;-) We'd save quite a bit of code by making that work properly.

Reply:
Indeed, I agree with you but the problem it is the `compute` method. 😕

Reply:
Hello @odony , the reason why we made a compute sudo instead of a related is to prevent the cache issue. Do you see any other way to achieve this? Thanks 🙂